internal/util/feature_flag.go

```diff
@@ -0,0 +1,17 @@
+package util
+
+var features = map[string][]string{
+	"dev":        {"assurance_visits"},
+	"production": {},
+}
+
+func FeatureFlag(env string) func(string) string {
+	return func(f string) string {
+		for _, v := range features[env] {
+			if v == f {
+				return ""
+			}
+		}
+		return "hide"
```

Comment:
I'd probably make this return a bool if a feature is set and check for it in the template. So css and features don't get too intertwined.